api_tests/src/post.spec.ts

```diff
@@ -225,6 +227,24 @@ test("Sticky a post", async () => {
   expect(betaPost3?.post.featured_community).toBe(false);
 });
 
+
+test("Collection of featured posts gets federated", async () => {
+  // create a new community and feature a post
+  let community = await createCommunity(alpha);
+  let post = await createPost(alpha,community.community_view.community.id);
+  let featuredPost = await featurePost(alpha, true, post.post_view.post);
+  expect(featuredPost.post_view.post.featured_community).toBe(true);
+
+  // fetch the community, ensure that post is also fetched and marked as featured
+  let betaCommunity = (await resolveCommunity(beta, community.community_view.community.actor_id));
+
+  const betaPost = await waitForPost(
+    beta,
+    post.post_view.post,
+    post => post?.post.featured_community === true,
+  );
```

Comment:
Probably should add an expect here.

Reply:
waitUntil already throws an error if it doesnt get a matching item after max retries.